composer.json

```diff
@@ -18,8 +18,8 @@
     "require-dev": {
         "phpunit/phpunit": "^8.4",
         "php-amqplib/php-amqplib": "^2.1",
-        "doctrine/common": "^2.3",
-        "doctrine/dbal": "^2.4",
+        "doctrine/common": "^3.0",
```

Comment:
this should not force the 3.0 version IMO, as the latest 2.x versions of common are still compatible. This should be using a `||` in the constraint.

Reply:
@lepiaf has changed this to 2.9, maybe we can allow 3.0 in Swarrot & SwarrotBundle in other MRs

Reply:
@stof is it OK for you?